docs/overview.md

```diff
@@ -6,7 +6,7 @@
     interact with Git
  - `IGitResult` - the abstraction for a result returned by Git - contains
     exit code and standard output/error text
- - `IGitExecutionOptions` - additional overrides to change the behaviour
+ - `IGitExecutionOptions` - additional overrides to change the behavior
```

Comment:
I've been writing the docs in British English wherever possible (not being 🇺🇸 ) - would you mind undoing this change? If there's wild inconsistencies let's address this separately.